packages/webcomponents/src/components/payments-list/payments-list.tsx

```diff
@@ -3,6 +3,7 @@ import { PaymentService } from '../../api/services/payment.service';
 import { makeGetPayments } from './get-payments';
 import { ErrorState } from '../details/utils';
 import { ComponentError, ComponentErrorCodes, ComponentErrorSeverity } from '../../api/ComponentError';
+import JustifiAnalytics from '../../api/Analytics';
 
```

Comment:
Not sure where is best to leave this comment - based on the description of the PR I am expecting every event emitted by a component to trigger a post to the `analytics` endpoint. 

I am triggering the `clickEvent` for clicking a payment row - the event is captured and logged in storybook Actions, but there is no analytics post when this event occurs. Is this intentional? Or should this be adjusted?

Reply:
Same applies to the click event for `payouts-list` as well

Reply:
I am not sure either what would be the ideal behavior on this.

Based on the current logic, the Analytics class will only track the events emitted by the component where the Analytics was initiated. Hence the click-event is emitted by a lower level component, the Analytics is not aware of that.

This was the easiest implementation but we absolutely can change this logic with a little bit more code if we decide to track all events down the tree. WDYT @jakemer @colganmeanor

Reply:
I think we can resolve this thread - we discussed this morning and agreed that the click events dont matter, at least right now for this PR.